looprpc/client.proto

```diff
@@ -419,7 +419,22 @@ message SwapInfoRequest {
 message TermsRequest {
 }
 
-message TermsResponse {
+message InTermsResponse {
+
+    reserved 1, 2, 3, 4, 7;
```

Comment:
This is to make it compatible with the old `TermsResponse`? Alternatively you could use a `oneof` and loop type specific sub messages. (I'm slightly in favor of the split though)

Reply:
Yes, to keep it backwards compatible at runtime.